### ISSUE TITLE
Boolean Question - When selecting a value for the first time, the Yes label is not announced by a narrator

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#9734]
T22769 - Screenreader issues
https://surveyjs.answerdesk.io/internal/ticket/details/T22769
***
```
{
  "pages": [
    {
      "name": "page1",
      "elements": [
        {
          "type": "text",
          "name": "question3"
        },
        {
          "type": "boolean",
          "name": "question4"
        },       
      ]
    }
  ]
}
```
https://github.com/user-attachments/assets/005aa70d-7f0a-4558-a76f-40a2ef6b4687
